### DISCUSSION TITLE
v3.13.0-beta.1

### DISCUSSION THREAD

**Discussion by FaithBeam** [FaithBeam/Sims-1-Complete-Collection-Widescreen-Patcher#117] [General]
This release adds these features to the **Extras** tab:
* Add weekends ([DomCal](https://white.leadpipecollection.net/?q=node/200#dd_domcal) mod by Damon)
  * Add the option to increase salaries by 30% to offset the money lost by not working on the weekends
* Add career editor

* [Add Weekends Wiki Page](https://github.com/FaithBeam/Sims-1-Complete-Collection-Widescreen-Patcher/wiki/Add-Weekends-(DomCal))
* [Career Editor Wiki Page](https://github.com/FaithBeam/Sims-1-Complete-Collection-Widescreen-Patcher/wiki/Career-Editor)

<hr /><em>This discussion was created from the release <a href='https://github.com/FaithBeam/Sims-1-Complete-Collection-Widescreen-Patcher/releases/tag/3.13.0-beta.1'>v3.13.0-beta.1</a>.</em>

**Comment by ghost** [comment]
Ohh I love this calendar I've had it for ages. I do note however. Not sure if your iff file has it but mine has typos in the description in buy mode and the text box that comes up when you click the calendar has typos too like they spelt holidays wrong haha so nice you're adding it to your launcher ❤️ just wanted to tell you.

**Reply by FaithBeam** [reply]
Thanks for pointing that out. I've fixed those typos in the latest beta. If you have any suggestions for other quality of life mods, let me know and I may include them assuming I get permission from the author.

**Comment by QueenRubyyy** [comment]
Sorry to ask here but i really want to edit my sims heads in Sims Legacy Collection, but i can't because the Cheat Unlocker/Widescreen Patcher doesn't work, will you update the Cheat Unlocker for the new release or no? It would be so lovely! Thanks in advance!!

**Reply by FaithBeam** [reply]
I'm working on enabling cheats in the Legacy Collection but I have no ETA right now.

**Reply by QueenRubyyy** [reply]
Wow awesome! I hope it goes well, take your time! I'm really looking forward to it!!

**Reply by ghost** [reply]
Oh yeah what are you guys thoughts on the Sims 1 and or Sims 2 legacy collection releases?
I bought the sims 1 legacy collection and installed it properly and it won't even launch. Just goes to load then poof it closes. Not even a window pops up haha. Bit crazy for an official release but it is EA lol.

**Comment by cryosyahA-a11y** [comment]
Hello, i just downloaded the Calender mod and am I supposed to put the file in the Download folder or somewhere else?

And where do I find it in the Buy Mode?
